src/handlers/frontend-auth/handler.ts

```diff
@@ -1,5 +1,51 @@
-import { APIGatewaySimpleAuthorizerResult } from "aws-lambda";
+import {
+  APIGatewayRequestAuthorizerEvent,
+  APIGatewayAuthorizerResult,
+} from "aws-lambda";
+import { randomUUID } from "crypto";
+const region = "eu-west-2";
+const accountId = process.env.AWS_ACCOUNT_ID;
+const verb = "GET";
 
-export const handler = async (): Promise<APIGatewaySimpleAuthorizerResult> => ({
-  isAuthorized: true,
+const generatePolicy = ({
+  apiId,
+  sub,
+  effect,
+  context,
+}: {
+  apiId: string;
+  sub: string;
+  effect: "Allow" | "Deny";
+  context?: Record<string, string | number | boolean | null | undefined>;
+}): APIGatewayAuthorizerResult => ({
+  principalId: sub,
+  policyDocument: {
+    Version: "2012-10-17",
+    Statement: [
+      {
+        Action: "execute-api:Invoke",
+        Effect: effect,
+        Resource: `arn:aws:execute-api:${region}:${accountId}:${apiId}/web/${verb}/*`,
```

Comment:
Running ui-tests in an environment with auth enabled will generally be tricky, but it would also be good to have a test or two for the login procedure.

Reply:
Auth is now disabled in all the envs where UI-tests run. PR-tests are passing.